### ISSUE TITLE
Redundant contact info?

### ISSUE THREAD

**Issue by dzerbino** [fairtracks/fairtracks_standard#44]
Contact information is required at both the collection and the study level, is that too much?

**Comment by sveinugu** [comment]
Yes, I think I agree. But the study level should perhaps have some requirements as to external links. What about that we require at least one of the following?:

- contact
- publication
- global_id

By the way:
- [ ] Publications should be using doi identifiers, not pubmed

**Comment by sveinugu** [comment]
Note that in my mind, a collection could also in theory be a "mixtape" of tracks from several sources gathered for a particular purpose by someone. Thus the contact info of the collection might have nothing to do with the study. In our case, I suppose the contact of the studies should not really be you (which might be why you raised the issue in the first place?).
